e2e/dapp-connections/WalletConnect.yaml

```diff
@@ -0,0 +1,82 @@
+appId: ${APP_ID}
+tags:
+  - browser
+  - parallel
+---
+- clearState
+- launchApp:
+    arguments:
+      isE2ETest: true
+- runFlow: ../utils/Prepare.yaml
+- runFlow: ../utils/ImportWalletWithKey.yaml
+
+# Navigate to browser
+- swipe:
+    direction: LEFT
+- assertVisible:
+    id: discover-header
+- swipe:
+    direction: LEFT
+- assertVisible:
+    id: browser-screen
+
+# Note: Since we can't directly interact with WebView content,
+# we use coordinates. These may need adjustment based on device.
+
+- runFlow:
+    when:
+      platform: Android
+    commands:
+      # connect to bx-test-dapp
+      # tap on browser search input
+      - tapOn:
+          point: 450, 1950
+      - assertVisible: 'Find apps and more'
+      - inputText: 'https://bx-test-dapp.vercel.app/'
+      - pressKey: 'Enter'
+      # tap on wallet pfp to open the control panel
+      - tapOn:
```

Comment:
I was playing with the malicious dapp test today and it seems possible to interact with webview elements on Android

Reply:
youre so right ty for the heads up